General/Playground.md

```diff
@@ -76,4 +81,86 @@ That could be useful if you want to understand the result value.
 
 ## Advance usage
 
->TODO
+In this section, we will cover some advance usages of the Playground. 
+
+![Overview of the Playground](Playground_overview.png)
+
+The previous figure highlight different parts of the playground we will refer to in this explanation.
+
+1. The playground name tab
+2. The run button
+3. The remote publishing button
+4. The bindings table
+5. The pages history
+
+Out of those, the simplest feature is the run button that will just execute the content of the playground and open an embedded inspector in the playground with the result of the expression. It is the equivalent of `Do it and go` (cmd/ctrl + g).
+
+### Variables bindings
+
+The playground is able to manage variables in two way. The first way is the classic Smalltalk way of declaring temporary variables (`| varName |`) at the beginning of the playground and using them later. 
+
+Example:
+
+```Smalltalk
+| result |
+result := 1 + 2.
+result := result + 3.
+```
+
+Using this kind of variables will work as the temporary variables of methods. During the execution they will be initialized at nil, they will be used during the current execution and then they will be discarded. 
+
+> Note: If you select a piece of code to execute it, do not forget to select the temporary variables declaration line.
+
+The second way to manage variables in the playground is to use them without declaring them. 
+
+Example:
+
+```Smalltalk
+result := 1 + 2.
+result := result + 3
+```
+
+By doing so, during its first use, the playground creates and initialize the new variable to nil and stores it. This means that the variable will still exist and will still contain its value after the execution. If you execute the previous piece of code in a playground and then select `result` to inspect it, you will find the value `6`.
+
+This feature is useful while playing with some code. For example, if you need to work on a model, instead of recreating the model at each run you can just initialize it one time and reuse the variable containing the model for all the playground session.
+
+If at some point you don't remember the content of some variables, you can find all the variables the playground bound in the binding table button (4 on the previous overview screenshot). 
+
```

Comment:
Maybe here you should also talk about how to unbind variables in the playground (the cross in the list of bound variables).